### PR TITLE
Us 218

### PR DESCRIPTION
**Pull request by BadamkhandB**
I finished this project give some advice

### CODE REVIEW

**Review by VeronikaKrasnokutska — CHANGES_REQUESTED** [review]
//add step 4, please

**Comment by ZaierAouani** [comment]
//add one million more steps lol

**Comment by Aramat16ukr** [comment]
good

File: src/US205/US_218/Checked.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package US205.US_218;
+
+public class Checked {
+ // step 1
+ // step 2
+ // step 3
```
Comment (AnnaLoshak) [inline src/US205/US_218/Checked.java:6]: could you please add one more step

**Review by AnnaLoshak — CHANGES_REQUESTED** [review]
could you please add one more step

**Review by Aramat16ukr — COMMENTED** [review]
add one step

**Review by Idemianiv — APPROVED** [review]
good

**Review by AnnaLoshak — APPROVED** [review]

**Review by VeronikaKrasnokutska — APPROVED** [review]

File: src/US205/US_218/Checked.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package US205.US_218;
+
+public class Checked {
```
Comment (LaylaNait) [inline src/US205/US_218/Checked.java:3]: add some methods

**Review by LaylaNait — CHANGES_REQUESTED** [review]
add methods

File: src/US205/US_218/Checked.java (hunk 2 above)
Comment (natebell510) [reply src/US205/US_218/Checked.java:3]: added review

**Review by LaylaNait — APPROVED** [review]
all done

File: src/US205/US_218/Checked.java (hunk 1 above)
Comment (natebell510) [reply src/US205/US_218/Checked.java:6]: add more steps

**Review by natebell510 — APPROVED** [review]

**Review by ZaierAouani — APPROVED** [review]

**Pull request merged**